### PR TITLE
chore(core): relax critical log level to error in attach/detach operations that do not need it

### PR DESCRIPTION
**Pull request by marregui**
DETACH_ERR_ACTIVE
DETACH_ERR_MISSING_PARTITION
DETACH_ERR_ALREADY_DETACHED
ATTACH_ERR_PARTITION_EXISTS

don't need to be critical

### CODE REVIEW

File: core/src/main/java/io/questdb/cairo/TableWriter.java [hunk 1]
```diff
@@ -6485,7 +6485,7 @@ static void indexAndCountDown(ColumnIndexer indexer, long lo, long hi, SOCountDo
             indexer.refreshSourceAndIndex(lo, hi);
         } catch (CairoException e) {
             indexer.distress();
-            LOG.critical().$("index error [fd=").$(indexer.getFd()).$(']').$('{').$((Sinkable) e).$('}').$();
+            LOG.critical().$("index error [fd=").$(indexer.getFd()).$("'{").$((Sinkable) e).$('}').$();
```
Comment (puzpuzpuz) [inline core/src/main/java/io/questdb/cairo/TableWriter.java:6488]:
```suggestion
            LOG.critical().$("index error [fd=").$(indexer.getFd()).$("]{").$((Sinkable) e).$('}').$();
```

**Comment by puzpuzpuz** [comment]
LGTM. Left a minor comment to address.

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 31 / 31 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/AttachDetachStatus.java|24|24|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|6|6|100.00%|

**Review by puzpuzpuz — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 31 / 31 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/AttachDetachStatus.java|24|24|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|6|6|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 31 / 31 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/AttachDetachStatus.java|24|24|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|6|6|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 31 / 31 (100.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/cairo/AttachDetachStatus.java|24|24|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|6|6|100.00%|

**Pull request merged**